### ISSUE TITLE
[BUG][opensearch] dead "minimum_master_nodes" setting still listed

### ISSUE THREAD

**Issue by rursprung** [opensearch-project/helm-charts#167]
**Describe the bug**
the opensearch chart still contains the commented-out `discovery.zen.minimum_master_nodes`:
https://github.com/opensearch-project/helm-charts/blob/12e6af67dd6e0e3945c9b3064579dd76381160d3/charts/opensearch/values.yaml#L50-L52

however, since Elasticsearch 7.0 this setting has been ignored: [release notes for 7.0](https://www.elastic.co/guide/en/elasticsearch/reference/7.10/breaking-changes-7.0.html#_cluster_bootstrapping_is_required_if_discovery_is_configured) (see also elastic/elasticsearch#37868)

accordingly this should probably be removed (unclear: should something else be added instead as a recommendation?)

**To Reproduce**
check the default config provided in the helm charts.

**Expected behavior**
only settings are listed which are still supported and which have an effect.

**Chart Name**
opensearch

**Screenshots**
n/a

**Host/Environment (please complete the following information):**
n/a

**Additional context**
i saw that this has also been touched upon in #54

**Comment by TheAlgo** [comment]
Yes this needs to be removed from the `values.yaml` as per my knowledge to avoid any confusion

**Comment by rursprung** [comment]
i'm not planning to contribute a PR here because i'm not familiar enough with what the proper settings would be for this use-case.
i'm actually hoping to see what you'll be doing :D

**Comment by peterzhuamazon** [comment]
@TheAlgo any progress?
Thanks.

**Comment by peterzhuamazon** [comment]
@TheAlgo Any updates?
Thanks.
